script/tasks/diamondSyncSigs_FAST.sh

```
@@ -0,0 +1,188 @@
+  FAILED_LOG_FILE=$(mktemp)
+
+  # Define function to process a single network
+  function processNetwork {
+    local NETWORK=$1
+
+    # Skip local/test networks
+    if [[ "$NETWORK" == "localanvil" || "$NETWORK" == "bsc-testnet" || "$NETWORK" == "lineatest" || "$NETWORK" == "mumbai" || "$NETWORK" == "sepolia" ]]; then
+      return
+    fi
+
+    local DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)